packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/components/offer-details/index.js

```diff
@@ -0,0 +1,12 @@
+import angular from 'angular';
+
+import component from './component';
+
+const moduleName = 'ovhManagerDedicatedCloudBackupOfferDetails';
+
+angular
+  .module(moduleName, ['oui'])
```

Comment:
`ovh-ui-angular` is not imported, neither are `angular-translate` and `ngTranslateAsyncLoader`

Reply:
Have added this